src/components/Card.js

```diff
@@ -25,41 +53,87 @@ export default class Card {
       .addEventListener('click', ()=> {
         this._handlePreviewDisplay();
       });
+  }*/
+
+  _showCardLikes() {
+    this._cardLikeCounter.textContent = this._likes.length;
+    if (this.cardIsLiked) {
+      this._cardLikeButton.classList.add("cards__like-button_active");
+    } else {
+      this._cardLikeButton.classList.remove("cards__like-button_active");
+    }
+  }
+
+  _renderLikes() {
+    if (this.isLiked) {
+      this._cardLikeButton.classList.add("cards__like-button_active");
+    } else {
+      this._cardLikeButton.classList.remove("cards__like-button_active");
+    }
+  }
+
+  updateLikes(isLiked) {
+    this.isLiked = isLiked;
+    this._renderLikes();
   }
 
   _handleLikeButton() {
```

Comment:
Needs correction: please, remove this method, since now you pass it via constructor arguments

Reply:
commented out